identity/app/views/consentJourney.scala.html

```diff
@@ -175,15 +175,35 @@ <h3 class="manage-account__switch-title">&nbsp;</h3>
 
 @confirmModal = {
     <section class="identity-forms-message">
-
+        @if(onlySubscribedToV1Newsletters.isEmpty) {
+            <h1 class="identity-header">Really?</h1>
```

Comment:
I find this tone a little aggressive personally - what's wrong with "Are you sure?"

Reply:
Celine suggested it. It was intentionally made quite strong. But I see how it may sound a bit patronising. Let's stick to 'Are you sure' for both modals.